internal/cmd/config.go

```diff
@@ -747,18 +762,33 @@ traefik:
 	v.SetDefault("cluster::posthook::autoscaler::enabled", true)
 
 	// v.SetDefault("cluster::disasterRecovery::enabled", true)
-	v.SetDefault("cluster::disasterRecovery::namespace", "")
+	v.SetDefault("cluster::disasterRecovery::namespace", "pipeline-system")
 	v.SetDefault("cluster::disasterRecovery::ark::syncEnabled", true)
 	v.SetDefault("cluster::disasterRecovery::ark::bucketSyncInterval", "10m")
 	v.SetDefault("cluster::disasterRecovery::ark::restoreSyncInterval", "20s")
 	v.SetDefault("cluster::disasterRecovery::ark::backupSyncInterval", "20s")
 	v.SetDefault("cluster::disasterRecovery::ark::restoreWaitTimeout", "5m")
-	v.SetDefault("cluster::disasterRecovery::charts::ark::chart", "banzaicloud-stable/ark")
-	v.SetDefault("cluster::disasterRecovery::charts::ark::version", "1.2.3")
+	v.SetDefault("cluster::disasterRecovery::charts::ark::chart", "banzaicloud-stable/velero")
```

Comment:
We should probably rename at least the chart to velero....in a subsequent PR. Please open an issue to track it.

Reply:
https://github.com/banzaicloud/pipeline/issues/3270